src/main/java/com/visenze/visearch/BaseSearchParams.java

```diff
@@ -28,7 +28,8 @@
     private static final Boolean DEFAULT_GET_ALL_FL = false;
     private static final Boolean DEFAULT_DEDUP = false;
     private static final Map<String, String> DEFAULT_VS_CONFIG = new HashMap<String, String>();
-
+    private static final List<DiversityQuery> DEFAULT_DIVERSITY_QUERIES = Lists.newArrayList();
+    public static final String DIVERSITY = "diversity";
```

Comment:
Moving this to `ViSearchHttpConstants` should be better

Reply:
i have refactored